sharkiq/ayla_api.py

```diff
@@ -75,7 +75,7 @@ def _set_credentials(self, status_code: int, login_result: Dict):
         self._access_token = login_result["access_token"]
         self._refresh_token = login_result["refresh_token"]
         self._auth_expiration = datetime.now() + timedelta(seconds=login_result["expires_in"])
-        self._is_authed = True
+        self._is_authed = True  # TODO: Any non 200 status code should cause this to be false
```

Comment:
This method explicitly checks for two specific error codes and assumes any other response was successful.

Reply:
I have moved this to issue #11 for the time being so that I can merge this